packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import colors from '../colors';
+
```

Comment:
Should we leave a TODO?

```suggestion
// TODO extend import { createTheme, ThemeOptions, Theme } from '@mui/system';

```